### DISCUSSION TITLE
考虑在对话记本中增加指定消息的回复功能吗？

### DISCUSSION THREAD

**Discussion by NightRain233** [WatsonYao/nian_app#58] [Q&A] [answered]
非常感谢作者大大的开发

**Comment by WatsonYao** [comment]
> 非常感谢作者大大的开发

是说的类似微信QQ里面的引用、回复这类的功能吗？

**Reply by NightRain233** [reply]
是的

**Reply by WatsonYao** [reply]
这边加到待办里面评估看看

**Comment by WatsonYao** [comment, marked as answer]
本着轻量的原则，对话本的核心还是左右文本的展示。
引用&回复这样的功能，涉及省略显示和点开全部展开，本身没有现在对话完全展示来的方便。
实在需要引用的话，可以复制之前的放到左侧，再在右侧答复。
如果需要大量的引用对话，还是建议用标准记本的回复功能，还没有数量的限制。
所以对话本的引用&回复功能，暂时不会安排开发计划。
